test/popup.spec.js

```diff
@@ -20,7 +20,7 @@ module.exports.describe = function({testRunner, expect, playwright, CHROMIUM, WE
   const {beforeAll, beforeEach, afterAll, afterEach} = testRunner;
 
   describe('window.open', function() {
-    it.fail(CHROMIUM)('should inherit user agent from browser context', async function({browser, server}) {
+    it('should inherit user agent from browser context', async function({browser, server}) {
```

Comment:
Let's add a similar test for anchor click.

Reply:
Done.